docs/advanced-features/middleware.md

```diff
@@ -93,9 +93,9 @@ export const config = {
 Configured matchers:
 
 1. MUST start with `/`
-1. can include named parameters: `/about/:path` matches `/about/a` and `/about/b` but not `/about/a/c`
-1. can have modifiers on named parameters (starting with `:`): `/about/:path*` matches `/about/a/b/c` because `*` is _zero or more_. `?` is _zero or one_ and `+` _one or more_
-1. can use regular expression enclosed in parenthesis: `/about/(.*)` is the same as `/about/:path*`
+2. can include named parameters: `/about/:path` matches `/about/a` and `/about/b` but not `/about/a/c`
+3. can have modifiers on named parameters (starting with `:`): `/about/:path*` matches `/about/a/b/c` because `*` is _zero or more_. `?` is _zero or one_ and `+` _one or more_
+4. can use regular expression enclosed in parenthesis: `/about/(.*)` is the same as `/about/:path*`
```

Comment:
```suggestion
2. Can include named parameters: `/about/:path` matches `/about/a` and `/about/b` but not `/about/a/c`
3. Can have modifiers on named parameters (starting with `:`): `/about/:path*` matches `/about/a/b/c` because `*` is _zero or more_. `?` is _zero or one_ and `+` _one or more_
4. Can use regular expression enclosed in parenthesis: `/about/(.*)` is the same as `/about/:path*`
```